app/models/crop.rb

```diff
@@ -21,16 +22,18 @@ class Crop < ActiveRecord::Base
   before_destroy {|crop| crop.posts.clear}
 
   default_scope { order("lower(name) asc") }
```

Comment:
I wasn't sure exactly how to do this. If I do this 
```
default_scope { where(:approval_status => "approved").reorder("created_at desc")  }
scope :pending_approval, -> { where(:approval_status => "pending") }
```

I get queries like this:

`SELECT "crops".* FROM "crops"  WHERE "crops"."approval_status" = 'approved' AND "crops"."approval_status" = 'pending'  ORDER BY lower(name) asc`